R/antigen.garnish_fitness.R

```diff
@@ -252,44 +261,39 @@ lapply(dtl %>% seq_along, function(i){
 
   # run pipeline
 
-  py_path <- system.file(package = "antigen.garnish") %>% file.path(., "extdata/src/main.py")
+    py_path <- system.file(package = "antigen.garnish") %>% file.path(., "extdata/src/main.py")
 
-  system(
+    on <- paste(nmerl, "_neoantigens_fitness_model_output.txt", sep = "")
+
+    system(
     paste("python",
           py_path,
           "neoepitopes.txt",
-          "./fitness_model",
+          dn,
           a,
           k,
-          "neoantigens_fitness_model_output.txt"
+          on,
+          nmerl
+    )
     )
-  )
 
   # curate output
-  if (!file.exists("neoantigens_fitness_model_output.txt")) {
-    warning("garnish_fitness did not return output.")
-    return(dt)
-  }
+    if (!file.exists(on)) {
+    warning(paste("garnish_fitness did not return output for nmers of length", nmerl, sep = " "))
+    return(NULL)
+    }
 
-  dto <- "neoantigens_fitness_model_output.txt" %>% data.table::fread
+    dto <- on %>% data.table::fread
 
-  dto %>% data.table::setnames(c("Mutation", "Sample"),
+    dto %>% data.table::setnames(c("Mutation", "Sample"),
                                c("var_uuid", "sample_id"))
-  dto %<>% melt(measure.vars = c("WildtypePeptide", "MutantPeptide"),
+    dto %<>% melt(measure.vars = c("WildtypePeptide", "MutantPeptide"),
                 value.name = "nmer")
 
-  # remove output columns before merging if they already exist
```

Comment:
Removed this, no longer necessary.  relic from when garnish_fitness was not called internally by garnish_predictions